tests/cli/test_version.py

```diff
@@ -62,7 +62,7 @@ def test_correct_output_ephemeral_sqlite(monkeypatch):
 Built:               {built.to_day_datetime_string()}
 OS/Arch:             {sys.platform}/{platform.machine()}
 Profile:             {profile.name}
-Server type:         ephemeral
+Server type:         EPHEMERAL
```

Comment:
@madkinsz We don't have any reason to believe that the lower-cased string values were in some way load-bearing do we?

Reply:
I can't think of any! Although it does seem nicer lowercase in this command output :)

Reply:
I can make it lowercase.